config/standalone.example.toml

```diff
@@ -177,3 +177,15 @@ parallel_scan_channel_size = 32
 # otlp_endpoint = "localhost:4317"
 # The percentage of tracing will be sampled and exported. Valid range `[0, 1]`, 1 means all traces are sampled, 0 means all traces are not sampled, the default value is 1. ratio > 1 are treated as 1. Fractions < 0 are treated as 0
 # tracing_sample_ratio = 1.0
+
+# standalone/frontend/datanode/metasrv export the metrics generated by itself
+# send metrics to Prometheus remote-write compatible receiver (e.g. `greptimedb`)
```

Comment:
- mention standalone/frontend/datanode/metasrv in their own config example
- add this section to each config file separately
- emphasize that the prometheus metric / otlp tracing are controlled by other options, and this is only for system metric